lxd/cgroup/abstraction.go

```diff
@@ -374,7 +419,7 @@ func (cg *CGroup) GetMemorySwapMaxUsage() (int64, error) {
 
 		swapValInt, err := strconv.ParseInt(swapVal, 10, 64)
 		if err != nil {
-			return -1, err
+			return -1, fmt.Errorf("Failed parsing %q: %w", swapVal, err)
```

Comment:
Why isn't `0` returned here like in the other changes? Shouldn't we make this consistent?